statistics/src/main/java/com/duckduckgo/app/statistics/api/featureusage/FeatureSegmentType.kt

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Overall would have been nicer to extract the segment stuff into its own module

Reply:
I thought about it but this is temporal, it will be live around 7-14 days

Reply:
Even if is temporary, the setup for a new module is trivial so there are no good reasons for not doing it right and instead add tech debt because we don't know if we will have to change the experiment or keep it for longer, etc. Plus when the time comes to delete the code you can delete the module altogether rather than searching for all the pieces.